src/components/feedback/Dialog.tsx

```diff
@@ -216,7 +215,7 @@ const Dialog = function Dialog({
           buttons={buttons}
           fullWidthHeader={true}
           icon={icon}
-          onClose={closeHandler}
+          onClose={isClosableDialog ? closeHandler : undefined}
```

Comment:
I could just have done `onClose ? closeHandler : undefined`, but I feel starting with `isClosableDialog` explains better the "why".

Reply:
I think that's fine.